keyless/pepper/service/src/dedicated_handlers/handlers.rs

```diff
@@ -41,10 +44,64 @@ pub trait HandlerTrait<TRequest, TResponse>: Send + Sync {
         jwk_cache: JWKCache,
         cached_resources: CachedResources,
         request: TRequest,
+        account_recovery_managers: Arc<AccountRecoveryManagers>,
         account_recovery_db: Arc<dyn AccountRecoveryDBInterface + Send + Sync>,
     ) -> Result<TResponse, PepperServiceError>;
 }
 
+/// A handler for processing pepper fetch requests
+pub struct V0DelegatedFetchHandler;
+
+#[async_trait]
+impl HandlerTrait<PepperRequestV2, PepperResponse> for V0DelegatedFetchHandler {
```

Comment:
nit: V0Handler handling RequestV2 sounds a bit confusing 😆

Reply:
Yeah, `v0` is for the API version; `PepperRequestV2` is because `PepperRequestV1` is insufficient. 😆 

Should be fine for now (at least the ugliness is contained inside `v0` of the API. We can always design and deploy a new version (`v1`) if we think it makes sense 😄